File: src/main/java/net/minecord/beautyindicator/command/ReloadCommand.java
package net.minecord.beautyindicator.command;

import net.minecord.beautyindicator.BeautyIndicator;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class ReloadCommand implements CommandExecutor {
    private BeautyIndicator beautyIndicator;

    public ReloadCommand(BeautyIndicator beautyIndicator) {
        this.beautyIndicator = beautyIndicator;
    }

    @Override
    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] strings) {
        if(commandSender.hasPermission("beautyindicator.reload") && strings.length > 0 && strings[0].equals("reload")) {
            beautyIndicator.onReload();
            commandSender.sendMessage(ChatColor.translateAlternateColorCodes('&', "&b[BeautyIndicator] &aPlugin was successfully reloaded"));
            return true;
        }
        return false;
    }
}

File: src/main/kotlin/net/minecord/beautyindicator/command/ReloadCommand.kt
package net.minecord.beautyindicator.command

import net.minecord.beautyindicator.BeautyIndicator
import net.minecord.beautyindicator.colored
import org.bukkit.ChatColor
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class ReloadCommand(private val beautyIndicator: BeautyIndicator) : CommandExecutor {
    override fun onCommand(commandSender: CommandSender, command: Command, s: String, strings: Array<String>): Boolean {
        if (commandSender.hasPermission("beautyindicator.reload") && strings.isNotEmpty() && strings[0] == "reload") {
            beautyIndicator.onReload()
            commandSender.sendMessage("&b[BeautyIndicator] &aPlugin was successfully reloaded".colored())
            return true
        }
        return false
    }
}
